internal/config/config_test.go

```diff
@@ -34,6 +34,11 @@ func TestNew(t *testing.T) {
 					Port:             8080,
 					AssetsDir:        "assets",
 					DataSyncInterval: 5 * time.Minute,
+					CORSConfig: cors.Options{
+						AllowedOrigins: []string{"*"},
+						AllowedMethods: []string{"GET", "POST", "PUT", "DELETE"},
```

Comment:
Do we have DELETE and PUT endpoints?

Reply:
Yeah, I think we can remove those. We only do GET.